### ISSUE TITLE
Offer to install updates at the end of install, before reboot

### ISSUE THREAD

**Issue by stormi** [xcp-ng/xcp#353]
In order to avoid one extra reboot, we could offer to install the latest updates at the end of the installation process, if the online repositories can be reached.

**Comment by Forza-tng** [comment]
> In order to avoid one extra reboot, we could offer to install the latest updates at the end of the installation process, if the online repositories can be reached.

IMHO this could be a good thing. However, it might be wrong to do a full update if the host is joining a pool that doesn't have the latest patches yet.

**Comment by olivierlambert** [comment]
Yeah, it could be a window asking "Do you want to download the last updates" or a checkbox before install asking for it. This way, it's still opt-in.
